bin/ChakraCore/ChakraCoreDllFunc.cpp

```diff
@@ -19,26 +19,8 @@
 #include "Base/EtwTrace.h"
 #endif
 
-#ifdef __APPLE__
-// dummy usage of JSRT to force export JSRT on dylib
-#include "ChakraCore.h"
-void DummyJSRTCall() {
-    JsRuntimeHandle *runtime;
-    JsRuntimeAttributes attr;
-    JsCreateRuntime(attr, nullptr, runtime);
-}
-#endif
-
 extern HANDLE g_hInstance;
-#ifdef _WIN32
 static ATOM  lockedDll = 0;
-#endif
-
-#ifdef _MSC_VER
-#define EXPORT_FUNC
```

Comment:
The removal of this break nojit builds. At the end of this file there are 2 functions under `#if !ENABLE_NATIVE_CODEGEN` that are still using `EXPORT_FUNC`. I am not sure what the exact fix is, could you take a look please